entities/vectorindex/hnsw/config.go

```diff
@@ -24,6 +24,7 @@ import (
 const (
 	DistanceCosine    = "cosine"
 	DistanceDot       = "dot"
+	DistanceGeo       = "geo"
```

Comment:
This case needs to be also taken into account in `config_handler.go:119` in `validateDefaultVectorDistanceMetric` method.`hnsw.DistanceGeo` case needs to be added there so that it could pass the validation